met-web/src/components/tenantManagement/Listing.tsx

```diff
@@ -0,0 +1,114 @@
+import { AddCircleOutline } from '@mui/icons-material';
+import { Box, Grid, Skeleton } from '@mui/material';
+import { Button } from 'components/MetDesignSystem/Input/Button';
+import { Header1, Header2, BodyText } from 'components/MetDesignSystem/Typography/';
+import { ResponsiveContainer } from 'components/MetDesignSystem/Layout';
+import {
+    Table,
+    TableBody,
+    TableCell,
+    TableHead,
+    TableHeadCell,
+    TableHeadRow,
+    TableRow,
+} from 'components/MetDesignSystem/Layout/Table';
+import { getAllTenants } from 'services/tenantService';
+
+import React, { useEffect } from 'react';
+import { Tenant } from 'models/tenant';
+import { Else, If, Then } from 'react-if';
+import { BreadcrumbTrail } from 'components/MetDesignSystem/Navigation/Breadcrumb';
+
+const TenantListing = () => {
+    const [tenants, setTenants] = React.useState<Tenant[]>([]);
+    const [loading, setLoading] = React.useState<boolean>(true);
+
+    useEffect(() => {
+        const fetchTenants = async () => {
+            const tenants = await getAllTenants();
```

Comment:
You might be able to save a couple lines of code(plus add more specific error handling), and the function declaration with:

```suggestion
            getAllTenants
            .then(returnedTenants => setTenants(returnedTenants)
            .catch(// Handle Error)
            .finally(() => setLoading(false))
```

Reply:
Unfortunately, `.finally()` isn't available because we're [compiling in ES2017 mode](https://github.com/bcgov/met-public/blob/fc97bcb2481bdf39b9b795172d2254efb8b74303/met-web/tsconfig.json#L14)

Reply:
I believe you should be able to use it if you want to still. I think the `target` option just indicates which ES features are "downgraded" and which are left intact. 

https://www.typescriptlang.org/tsconfig/#target